2015-12-12 00:41:49,830 INFO [main] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: Created MRAppMaster for application appattempt_1449846414359_0019_000001
2015-12-12 00:41:50,308 WARN [main] org.apache.hadoop.util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
2015-12-12 00:41:50,327 INFO [main] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: Executing with tokens:
2015-12-12 00:41:50,327 INFO [main] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: Kind: YARN_AM_RM_TOKEN, Service: , Ident: (appAttemptId { application_id { id: 19 cluster_timestamp: 1449846414359 } attemptId: 1 } keyId: 823470771)
2015-12-12 00:41:50,486 INFO [main] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: Using mapred newApiCommitter.
2015-12-12 00:41:51,054 INFO [main] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: OutputCommitter set in config null
2015-12-12 00:41:51,121 INFO [main] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: OutputCommitter is org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter
2015-12-12 00:41:51,182 INFO [main] org.apache.hadoop.yarn.event.AsyncDispatcher: Registering class org.apache.hadoop.mapreduce.jobhistory.EventType for class org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler
2015-12-12 00:41:51,183 INFO [main] org.apache.hadoop.yarn.event.AsyncDispatcher: Registering class org.apache.hadoop.mapreduce.v2.app.job.event.JobEventType for class org.apache.hadoop.mapreduce.v2.app.MRAppMaster$JobEventDispatcher
2015-12-12 00:41:51,184 INFO [main] org.apache.hadoop.yarn.event.AsyncDispatcher: Registering class org.apache.hadoop.mapreduce.v2.app.job.event.TaskEventType for class org.apache.hadoop.mapreduce.v2.app.MRAppMaster$TaskEventDispatcher
2015-12-12 00:41:51,185 INFO [main] org.apache.hadoop.yarn.event.AsyncDispatcher: Registering class org.apache.hadoop.mapreduce.v2.app.job.event.TaskAttemptEventType for class org.apache.hadoop.mapreduce.v2.app.MRAppMaster$TaskAttemptEventDispatcher
2015-12-12 00:41:51,185 INFO [main] org.apache.hadoop.yarn.event.AsyncDispatcher: Registering class org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventType for class org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler
2015-12-12 00:41:51,199 INFO [main] org.apache.hadoop.yarn.event.AsyncDispatcher: Registering class org.apache.hadoop.mapreduce.v2.app.speculate.Speculator$EventType for class org.apache.hadoop.mapreduce.v2.app.MRAppMaster$SpeculatorEventDispatcher
2015-12-12 00:41:51,200 INFO [main] org.apache.hadoop.yarn.event.AsyncDispatcher: Registering class org.apache.hadoop.mapreduce.v2.app.rm.ContainerAllocator$EventType for class org.apache.hadoop.mapreduce.v2.app.MRAppMaster$ContainerAllocatorRouter
2015-12-12 00:41:51,201 INFO [main] org.apache.hadoop.yarn.event.AsyncDispatcher: Registering class org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncher$EventType for class org.apache.hadoop.mapreduce.v2.app.MRAppMaster$ContainerLauncherRouter
2015-12-12 00:41:51,261 INFO [main] org.apache.hadoop.mapreduce.v2.jobhistory.JobHistoryUtils: Default file system [hdfs://localhost:9000]
2015-12-12 00:41:51,294 INFO [main] org.apache.hadoop.mapreduce.v2.jobhistory.JobHistoryUtils: Default file system [hdfs://localhost:9000]
2015-12-12 00:41:51,320 INFO [main] org.apache.hadoop.mapreduce.v2.jobhistory.JobHistoryUtils: Default file system [hdfs://localhost:9000]
2015-12-12 00:41:51,333 INFO [main] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: Emitting job history data to the timeline server is not enabled
2015-12-12 00:41:51,383 INFO [main] org.apache.hadoop.yarn.event.AsyncDispatcher: Registering class org.apache.hadoop.mapreduce.v2.app.job.event.JobFinishEvent$Type for class org.apache.hadoop.mapreduce.v2.app.MRAppMaster$JobFinishEventHandler
2015-12-12 00:41:51,688 INFO [main] org.apache.hadoop.metrics2.impl.MetricsConfig: loaded properties from hadoop-metrics2.properties
2015-12-12 00:41:51,765 INFO [main] org.apache.hadoop.metrics2.impl.MetricsSystemImpl: Scheduled snapshot period at 10 second(s).
2015-12-12 00:41:51,765 INFO [main] org.apache.hadoop.metrics2.impl.MetricsSystemImpl: MRAppMaster metrics system started
2015-12-12 00:41:51,774 INFO [main] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: Adding job token for job_1449846414359_0019 to jobTokenSecretManager
2015-12-12 00:41:51,949 INFO [main] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: Not uberizing job_1449846414359_0019 because: not enabled;
2015-12-12 00:41:51,962 INFO [main] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: Input size for job job_1449846414359_0019 = 21770535. Number of splits = 3
2015-12-12 00:41:51,964 INFO [main] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: Number of reduces for job job_1449846414359_0019 = 1
2015-12-12 00:41:51,964 INFO [main] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: job_1449846414359_0019Job Transitioned from NEW to INITED
2015-12-12 00:41:51,965 INFO [main] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: MRAppMaster launching normal, non-uberized, multi-container job job_1449846414359_0019.
2015-12-12 00:41:51,996 INFO [main] org.apache.hadoop.ipc.CallQueueManager: Using callQueue class java.util.concurrent.LinkedBlockingQueue
2015-12-12 00:41:52,009 INFO [Socket Reader #1 for port 58617] org.apache.hadoop.ipc.Server: Starting Socket Reader #1 for port 58617
2015-12-12 00:41:52,040 INFO [main] org.apache.hadoop.yarn.factories.impl.pb.RpcServerFactoryPBImpl: Adding protocol org.apache.hadoop.mapreduce.v2.api.MRClientProtocolPB to the server
2015-12-12 00:41:52,040 INFO [IPC Server Responder] org.apache.hadoop.ipc.Server: IPC Server Responder: starting
2015-12-12 00:41:52,041 INFO [IPC Server listener on 58617] org.apache.hadoop.ipc.Server: IPC Server listener on 58617: starting
2015-12-12 00:41:52,041 INFO [main] org.apache.hadoop.mapreduce.v2.app.client.MRClientService: Instantiated MRClientService at ubuntu/127.0.1.1:58617
2015-12-12 00:41:52,143 INFO [main] org.mortbay.log: Logging to org.slf4j.impl.Log4jLoggerAdapter(org.mortbay.log) via org.mortbay.log.Slf4jLog
2015-12-12 00:41:52,146 INFO [main] org.apache.hadoop.http.HttpRequestLog: Http request log for http.requests.mapreduce is not defined
2015-12-12 00:41:52,159 INFO [main] org.apache.hadoop.http.HttpServer2: Added global filter 'safety' (class=org.apache.hadoop.http.HttpServer2$QuotingInputFilter)
2015-12-12 00:41:52,168 INFO [main] org.apache.hadoop.http.HttpServer2: Added filter AM_PROXY_FILTER (class=org.apache.hadoop.yarn.server.webproxy.amfilter.AmIpFilter) to context mapreduce
2015-12-12 00:41:52,168 INFO [main] org.apache.hadoop.http.HttpServer2: Added filter AM_PROXY_FILTER (class=org.apache.hadoop.yarn.server.webproxy.amfilter.AmIpFilter) to context static
2015-12-12 00:41:52,173 INFO [main] org.apache.hadoop.http.HttpServer2: adding path spec: /mapreduce/*
2015-12-12 00:41:52,173 INFO [main] org.apache.hadoop.http.HttpServer2: adding path spec: /ws/*
2015-12-12 00:41:52,183 INFO [main] org.apache.hadoop.http.HttpServer2: Jetty bound to port 57283
2015-12-12 00:41:52,183 INFO [main] org.mortbay.log: jetty-6.1.26
2015-12-12 00:41:52,235 INFO [main] org.mortbay.log: Extract jar:file:/home/parallels/Development/hadoop-2.6.0/share/hadoop/yarn/hadoop-yarn-common-2.6.0.jar!/webapps/mapreduce to /tmp/Jetty_0_0_0_0_57283_mapreduce____.a174mq/webapp
2015-12-12 00:41:52,578 INFO [main] org.mortbay.log: Started HttpServer2$SelectChannelConnectorWithSafeStartup@0.0.0.0:57283
2015-12-12 00:41:52,578 INFO [main] org.apache.hadoop.yarn.webapp.WebApps: Web app /mapreduce started at 57283
2015-12-12 00:41:52,983 INFO [main] org.apache.hadoop.yarn.webapp.WebApps: Registered webapp guice modules
2015-12-12 00:41:52,986 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: JOB_CREATE job_1449846414359_0019
2015-12-12 00:41:52,988 INFO [main] org.apache.hadoop.ipc.CallQueueManager: Using callQueue class java.util.concurrent.LinkedBlockingQueue
2015-12-12 00:41:52,989 INFO [Socket Reader #1 for port 33520] org.apache.hadoop.ipc.Server: Starting Socket Reader #1 for port 33520
2015-12-12 00:41:52,992 INFO [IPC Server Responder] org.apache.hadoop.ipc.Server: IPC Server Responder: starting
2015-12-12 00:41:52,992 INFO [IPC Server listener on 33520] org.apache.hadoop.ipc.Server: IPC Server listener on 33520: starting
2015-12-12 00:41:53,013 INFO [main] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: nodeBlacklistingEnabled:true
2015-12-12 00:41:53,013 INFO [main] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: maxTaskFailuresPerNode is 3
2015-12-12 00:41:53,013 INFO [main] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: blacklistDisablePercent is 33
2015-12-12 00:41:53,106 INFO [main] org.apache.hadoop.yarn.client.RMProxy: Connecting to ResourceManager at /0.0.0.0:8030
2015-12-12 00:41:53,270 INFO [main] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: maxContainerCapability: <memory:8192, vCores:32>
2015-12-12 00:41:53,270 INFO [main] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: queue: default
2015-12-12 00:41:53,290 INFO [main] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Upper limit on the thread pool size is 500
2015-12-12 00:41:53,296 INFO [main] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: yarn.client.max-cached-nodemanagers-proxies : 0
2015-12-12 00:41:53,339 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: job_1449846414359_0019Job Transitioned from INITED to SETUP
2015-12-12 00:41:53,351 INFO [CommitterEvent Processor #0] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: JOB_SETUP
2015-12-12 00:41:53,377 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: job_1449846414359_0019Job Transitioned from SETUP to RUNNING
2015-12-12 00:41:53,418 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:41:53,426 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_m_000000 Task Transitioned from NEW to SCHEDULED
2015-12-12 00:41:53,427 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:41:53,427 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_m_000001 Task Transitioned from NEW to SCHEDULED
2015-12-12 00:41:53,427 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:41:53,427 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_m_000002 Task Transitioned from NEW to SCHEDULED
2015-12-12 00:41:53,428 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_r_000000 Task Transitioned from NEW to SCHEDULED
2015-12-12 00:41:53,432 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_0 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:41:53,432 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_0 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:41:53,432 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_0 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:41:53,432 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_r_000000_0 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:41:53,434 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: mapResourceRequest:<memory:1024, vCores:1>
2015-12-12 00:41:53,449 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: reduceResourceRequest:<memory:1024, vCores:1>
2015-12-12 00:41:53,491 INFO [eventHandlingThread] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: Event Writer setup for JobId: job_1449846414359_0019, File: hdfs://localhost:9000/tmp/hadoop-yarn/staging/parallels/.staging/job_1449846414359_0019/job_1449846414359_0019_1.jhist
2015-12-12 00:41:54,292 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Before Scheduling: PendingReds:1 ScheduledMaps:3 ScheduledReds:0 AssignedMaps:0 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:0 ContRel:0 HostLocal:0 RackLocal:0
2015-12-12 00:41:54,344 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=3 release= 0 newContainers=0 finishedContainers=0 resourcelimit=<memory:6144, vCores:0> knownNMs=1
2015-12-12 00:41:54,345 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:6144, vCores:0>
2015-12-12 00:41:54,345 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:41:55,368 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 3
2015-12-12 00:41:55,369 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000002 to attempt_1449846414359_0019_m_000000_0
2015-12-12 00:41:55,370 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000003 to attempt_1449846414359_0019_m_000001_0
2015-12-12 00:41:55,370 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000004 to attempt_1449846414359_0019_m_000002_0
2015-12-12 00:41:55,370 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:3072, vCores:-3>
2015-12-12 00:41:55,370 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:41:55,370 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:0 ScheduledReds:0 AssignedMaps:3 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:3 ContRel:0 HostLocal:3 RackLocal:0
2015-12-12 00:41:55,427 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:41:55,441 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: The job-jar file on the remote FS is hdfs://localhost:9000/tmp/hadoop-yarn/staging/parallels/.staging/job_1449846414359_0019/job.jar
2015-12-12 00:41:55,444 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: The job-conf file on the remote FS is /tmp/hadoop-yarn/staging/parallels/.staging/job_1449846414359_0019/job.xml
2015-12-12 00:41:55,449 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Adding #0 tokens and #1 secret keys for NM use for launching container
2015-12-12 00:41:55,449 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Size of containertokens_dob is 1
2015-12-12 00:41:55,449 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Putting shuffle token in serviceData
2015-12-12 00:41:55,486 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_0 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:41:55,490 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:41:55,490 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_0 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:41:55,491 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:41:55,491 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_0 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:41:55,496 INFO [ContainerLauncher #0] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000002 taskAttempt attempt_1449846414359_0019_m_000000_0
2015-12-12 00:41:55,501 INFO [ContainerLauncher #0] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000000_0
2015-12-12 00:41:55,502 INFO [ContainerLauncher #1] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000003 taskAttempt attempt_1449846414359_0019_m_000001_0
2015-12-12 00:41:55,502 INFO [ContainerLauncher #1] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000001_0
2015-12-12 00:41:55,505 INFO [ContainerLauncher #0] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:41:55,505 INFO [ContainerLauncher #2] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000004 taskAttempt attempt_1449846414359_0019_m_000002_0
2015-12-12 00:41:55,505 INFO [ContainerLauncher #2] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000002_0
2015-12-12 00:41:55,605 INFO [ContainerLauncher #1] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:41:55,606 INFO [ContainerLauncher #2] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:41:55,662 INFO [ContainerLauncher #1] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000001_0 : 13562
2015-12-12 00:41:55,663 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000001_0] using containerId: [container_1449846414359_0019_01_000003 on NM: [ubuntu:49665]
2015-12-12 00:41:55,665 INFO [ContainerLauncher #2] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000002_0 : 13562
2015-12-12 00:41:55,667 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_0 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:41:55,667 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000002_0] using containerId: [container_1449846414359_0019_01_000004 on NM: [ubuntu:49665]
2015-12-12 00:41:55,668 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_0 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:41:55,668 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000001
2015-12-12 00:41:55,668 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_m_000001 Task Transitioned from SCHEDULED to RUNNING
2015-12-12 00:41:55,669 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000002
2015-12-12 00:41:55,669 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_m_000002 Task Transitioned from SCHEDULED to RUNNING
2015-12-12 00:41:55,670 INFO [ContainerLauncher #0] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000000_0 : 13562
2015-12-12 00:41:55,670 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000000_0] using containerId: [container_1449846414359_0019_01_000002 on NM: [ubuntu:49665]
2015-12-12 00:41:55,671 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_0 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:41:55,671 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000000
2015-12-12 00:41:55,671 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_m_000000 Task Transitioned from SCHEDULED to RUNNING
2015-12-12 00:41:56,386 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=3 release= 0 newContainers=0 finishedContainers=0 resourcelimit=<memory:3072, vCores:-3> knownNMs=1
2015-12-12 00:42:00,343 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:00,345 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:00,366 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:00,396 INFO [IPC Server handler 2 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000003 asked for a task
2015-12-12 00:42:00,396 INFO [IPC Server handler 2 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000003 given task: attempt_1449846414359_0019_m_000001_0
2015-12-12 00:42:00,398 INFO [IPC Server handler 1 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000002 asked for a task
2015-12-12 00:42:00,398 INFO [IPC Server handler 1 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000002 given task: attempt_1449846414359_0019_m_000000_0
2015-12-12 00:42:00,399 INFO [IPC Server handler 3 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000004 asked for a task
2015-12-12 00:42:00,399 INFO [IPC Server handler 3 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000004 given task: attempt_1449846414359_0019_m_000002_0
2015-12-12 00:42:10,554 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:10,558 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:10,893 INFO [IPC Server handler 26 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Progress of TaskAttempt attempt_1449846414359_0019_m_000000_0 is : 0.0
2015-12-12 00:42:10,893 INFO [IPC Server handler 12 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Progress of TaskAttempt attempt_1449846414359_0019_m_000002_0 is : 0.0
2015-12-12 00:42:11,052 FATAL [IPC Server handler 5 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Task: attempt_1449846414359_0019_m_000002_0 - exited : java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:11,053 INFO [IPC Server handler 5 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Diagnostics report from attempt_1449846414359_0019_m_000002_0: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:11,062 FATAL [IPC Server handler 6 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Task: attempt_1449846414359_0019_m_000000_0 - exited : java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:11,068 INFO [IPC Server handler 6 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Diagnostics report from attempt_1449846414359_0019_m_000000_0: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:11,147 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000000_0: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:11,151 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_0 TaskAttempt Transitioned from RUNNING to FAIL_CONTAINER_CLEANUP
2015-12-12 00:42:11,151 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000002_0: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:11,152 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_0 TaskAttempt Transitioned from RUNNING to FAIL_CONTAINER_CLEANUP
2015-12-12 00:42:11,161 INFO [ContainerLauncher #3] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000002 taskAttempt attempt_1449846414359_0019_m_000000_0
2015-12-12 00:42:11,164 INFO [ContainerLauncher #4] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000004 taskAttempt attempt_1449846414359_0019_m_000002_0
2015-12-12 00:42:11,164 INFO [ContainerLauncher #3] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000000_0
2015-12-12 00:42:11,166 INFO [ContainerLauncher #4] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000002_0
2015-12-12 00:42:11,167 INFO [ContainerLauncher #3] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:11,181 INFO [ContainerLauncher #4] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:11,413 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_0 TaskAttempt Transitioned from FAIL_CONTAINER_CLEANUP to FAIL_TASK_CLEANUP
2015-12-12 00:42:11,413 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_0 TaskAttempt Transitioned from FAIL_CONTAINER_CLEANUP to FAIL_TASK_CLEANUP
2015-12-12 00:42:11,418 INFO [CommitterEvent Processor #2] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:11,418 INFO [CommitterEvent Processor #1] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:11,562 WARN [CommitterEvent Processor #1] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000000_0
2015-12-12 00:42:11,562 WARN [CommitterEvent Processor #2] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000002_0
2015-12-12 00:42:11,585 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000003
2015-12-12 00:42:11,589 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:4096, vCores:-2>
2015-12-12 00:42:11,590 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:11,591 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:0 ScheduledReds:0 AssignedMaps:2 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:3 ContRel:0 HostLocal:3 RackLocal:0
2015-12-12 00:42:11,610 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_0 TaskAttempt Transitioned from FAIL_TASK_CLEANUP to FAILED
2015-12-12 00:42:11,611 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_0 TaskAttempt Transitioned from FAIL_TASK_CLEANUP to FAILED
2015-12-12 00:42:11,615 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_0 TaskAttempt Transitioned from RUNNING to FAIL_CONTAINER_CLEANUP
2015-12-12 00:42:11,615 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000001_0: Container killed on request. Exit code is 137
Container exited with a non-zero exit code 137
Killed by external signal

2015-12-12 00:42:11,787 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:11,792 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:11,795 INFO [ContainerLauncher #5] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000003 taskAttempt attempt_1449846414359_0019_m_000001_0
2015-12-12 00:42:11,795 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: 1 failures on node ubuntu
2015-12-12 00:42:11,795 INFO [ContainerLauncher #5] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000001_0
2015-12-12 00:42:11,795 INFO [ContainerLauncher #5] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:11,803 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_1 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:42:11,803 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: 2 failures on node ubuntu
2015-12-12 00:42:11,804 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_1 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:42:11,807 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Added attempt_1449846414359_0019_m_000000_1 to list of failed maps
2015-12-12 00:42:11,812 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Added attempt_1449846414359_0019_m_000002_1 to list of failed maps
2015-12-12 00:42:11,821 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_0 TaskAttempt Transitioned from FAIL_CONTAINER_CLEANUP to FAIL_TASK_CLEANUP
2015-12-12 00:42:11,821 INFO [CommitterEvent Processor #3] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:11,825 WARN [CommitterEvent Processor #3] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000001_0
2015-12-12 00:42:11,825 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_0 TaskAttempt Transitioned from FAIL_TASK_CLEANUP to FAILED
2015-12-12 00:42:11,826 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:11,826 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: 3 failures on node ubuntu
2015-12-12 00:42:11,826 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: Blacklisted host ubuntu
2015-12-12 00:42:11,826 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_1 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:42:11,827 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Added attempt_1449846414359_0019_m_000001_1 to list of failed maps
2015-12-12 00:42:12,591 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Before Scheduling: PendingReds:1 ScheduledMaps:3 ScheduledReds:0 AssignedMaps:2 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:3 ContRel:0 HostLocal:3 RackLocal:0
2015-12-12 00:42:12,651 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=0 finishedContainers=2 resourcelimit=<memory:6144, vCores:0> knownNMs=1
2015-12-12 00:42:12,651 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: Update the blacklist for application_1449846414359_0019: blacklistAdditions=1 blacklistRemovals=0
2015-12-12 00:42:12,651 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: Ignore blacklisting set to true. Known: 1, Blacklisted: 1, 100%
2015-12-12 00:42:12,651 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000002
2015-12-12 00:42:12,651 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000004
2015-12-12 00:42:12,651 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:6144, vCores:0>
2015-12-12 00:42:12,652 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000000_0: Container killed by the ApplicationMaster.
Container killed on request. Exit code is 143
Container exited with a non-zero exit code 143

2015-12-12 00:42:12,652 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:12,653 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:3 ScheduledReds:0 AssignedMaps:0 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:3 ContRel:0 HostLocal:3 RackLocal:0
2015-12-12 00:42:12,653 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000002_0: Container killed by the ApplicationMaster.
Container killed on request. Exit code is 143
Container exited with a non-zero exit code 143

2015-12-12 00:42:13,660 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: Update the blacklist for application_1449846414359_0019: blacklistAdditions=0 blacklistRemovals=1
2015-12-12 00:42:13,660 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:6144, vCores:0>
2015-12-12 00:42:13,660 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:14,680 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 1
2015-12-12 00:42:14,681 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigning container Container: [ContainerId: container_1449846414359_0019_01_000005, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] to fast fail map
2015-12-12 00:42:14,681 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned from earlierFailedMaps
2015-12-12 00:42:14,681 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000005 to attempt_1449846414359_0019_m_000000_1
2015-12-12 00:42:14,681 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:5120, vCores:-1>
2015-12-12 00:42:14,681 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:14,681 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:2 ScheduledReds:0 AssignedMaps:1 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:4 ContRel:0 HostLocal:3 RackLocal:0
2015-12-12 00:42:14,683 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:14,685 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_1 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:42:14,686 INFO [ContainerLauncher #6] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000005 taskAttempt attempt_1449846414359_0019_m_000000_1
2015-12-12 00:42:14,686 INFO [ContainerLauncher #6] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000000_1
2015-12-12 00:42:14,686 INFO [ContainerLauncher #6] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:14,725 INFO [ContainerLauncher #6] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000000_1 : 13562
2015-12-12 00:42:14,726 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000000_1] using containerId: [container_1449846414359_0019_01_000005 on NM: [ubuntu:49665]
2015-12-12 00:42:14,726 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_1 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:42:14,726 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000000
2015-12-12 00:42:15,697 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=1 finishedContainers=0 resourcelimit=<memory:4096, vCores:-2> knownNMs=1
2015-12-12 00:42:15,697 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 1
2015-12-12 00:42:15,697 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigning container Container: [ContainerId: container_1449846414359_0019_01_000006, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] to fast fail map
2015-12-12 00:42:15,697 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned from earlierFailedMaps
2015-12-12 00:42:15,698 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000006 to attempt_1449846414359_0019_m_000002_1
2015-12-12 00:42:15,698 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:4096, vCores:-2>
2015-12-12 00:42:15,698 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:15,698 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:1 ScheduledReds:0 AssignedMaps:2 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:5 ContRel:0 HostLocal:3 RackLocal:0
2015-12-12 00:42:15,698 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:15,699 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_1 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:42:15,700 INFO [ContainerLauncher #7] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000006 taskAttempt attempt_1449846414359_0019_m_000002_1
2015-12-12 00:42:15,700 INFO [ContainerLauncher #7] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000002_1
2015-12-12 00:42:15,700 INFO [ContainerLauncher #7] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:15,740 INFO [ContainerLauncher #7] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000002_1 : 13562
2015-12-12 00:42:15,740 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000002_1] using containerId: [container_1449846414359_0019_01_000006 on NM: [ubuntu:49665]
2015-12-12 00:42:15,741 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_1 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:42:15,741 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000002
2015-12-12 00:42:16,728 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=1 finishedContainers=0 resourcelimit=<memory:3072, vCores:-3> knownNMs=1
2015-12-12 00:42:16,728 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 1
2015-12-12 00:42:16,728 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigning container Container: [ContainerId: container_1449846414359_0019_01_000007, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] to fast fail map
2015-12-12 00:42:16,728 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned from earlierFailedMaps
2015-12-12 00:42:16,728 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000007 to attempt_1449846414359_0019_m_000001_1
2015-12-12 00:42:16,728 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:3072, vCores:-3>
2015-12-12 00:42:16,728 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:16,728 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:0 ScheduledReds:0 AssignedMaps:3 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:6 ContRel:0 HostLocal:3 RackLocal:0
2015-12-12 00:42:16,729 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:16,729 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_1 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:42:16,730 INFO [ContainerLauncher #8] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000007 taskAttempt attempt_1449846414359_0019_m_000001_1
2015-12-12 00:42:16,730 INFO [ContainerLauncher #8] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000001_1
2015-12-12 00:42:16,730 INFO [ContainerLauncher #8] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:16,813 INFO [ContainerLauncher #8] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000001_1 : 13562
2015-12-12 00:42:16,813 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000001_1] using containerId: [container_1449846414359_0019_01_000007 on NM: [ubuntu:49665]
2015-12-12 00:42:16,813 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_1 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:42:16,814 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000001
2015-12-12 00:42:17,739 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=1 finishedContainers=0 resourcelimit=<memory:2048, vCores:-4> knownNMs=1
2015-12-12 00:42:17,739 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 1
2015-12-12 00:42:17,740 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Cannot assign container Container: [ContainerId: container_1449846414359_0019_01_000008, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] for a map as either  container memory less than required <memory:1024, vCores:1> or no pending map tasks - maps.isEmpty=true
2015-12-12 00:42:17,740 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:2048, vCores:-4>
2015-12-12 00:42:17,740 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:17,740 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:0 ScheduledReds:0 AssignedMaps:3 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:7 ContRel:1 HostLocal:3 RackLocal:0
2015-12-12 00:42:18,394 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:18,421 INFO [IPC Server handler 3 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000005 asked for a task
2015-12-12 00:42:18,421 INFO [IPC Server handler 3 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000005 given task: attempt_1449846414359_0019_m_000000_1
2015-12-12 00:42:18,537 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:18,572 INFO [IPC Server handler 14 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000006 asked for a task
2015-12-12 00:42:18,573 INFO [IPC Server handler 14 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000006 given task: attempt_1449846414359_0019_m_000002_1
2015-12-12 00:42:18,759 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=0 release= 1 newContainers=0 finishedContainers=1 resourcelimit=<memory:3072, vCores:-3> knownNMs=1
2015-12-12 00:42:18,760 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000008
2015-12-12 00:42:18,760 ERROR [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Container complete event for unknown container id container_1449846414359_0019_01_000008
2015-12-12 00:42:18,760 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:3072, vCores:-3>
2015-12-12 00:42:18,760 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:20,657 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:20,669 INFO [IPC Server handler 12 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000007 asked for a task
2015-12-12 00:42:20,669 INFO [IPC Server handler 12 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000007 given task: attempt_1449846414359_0019_m_000001_1
2015-12-12 00:42:23,992 INFO [IPC Server handler 26 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Progress of TaskAttempt attempt_1449846414359_0019_m_000000_1 is : 0.0
2015-12-12 00:42:24,080 INFO [IPC Server handler 6 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Progress of TaskAttempt attempt_1449846414359_0019_m_000002_1 is : 0.0
2015-12-12 00:42:24,081 FATAL [IPC Server handler 5 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Task: attempt_1449846414359_0019_m_000000_1 - exited : java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:24,082 INFO [IPC Server handler 5 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Diagnostics report from attempt_1449846414359_0019_m_000000_1: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:24,099 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000000_1: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:24,101 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_1 TaskAttempt Transitioned from RUNNING to FAIL_CONTAINER_CLEANUP
2015-12-12 00:42:24,104 INFO [ContainerLauncher #9] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000005 taskAttempt attempt_1449846414359_0019_m_000000_1
2015-12-12 00:42:24,105 INFO [ContainerLauncher #9] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000000_1
2015-12-12 00:42:24,115 INFO [ContainerLauncher #9] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:24,373 FATAL [IPC Server handler 24 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Task: attempt_1449846414359_0019_m_000002_1 - exited : java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:24,387 INFO [IPC Server handler 24 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Diagnostics report from attempt_1449846414359_0019_m_000002_1: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:24,407 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000002_1: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:24,408 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_1 TaskAttempt Transitioned from RUNNING to FAIL_CONTAINER_CLEANUP
2015-12-12 00:42:24,435 INFO [ContainerLauncher #2] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000006 taskAttempt attempt_1449846414359_0019_m_000002_1
2015-12-12 00:42:24,450 INFO [ContainerLauncher #2] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000002_1
2015-12-12 00:42:24,452 INFO [ContainerLauncher #2] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:24,552 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_1 TaskAttempt Transitioned from FAIL_CONTAINER_CLEANUP to FAIL_TASK_CLEANUP
2015-12-12 00:42:24,553 INFO [CommitterEvent Processor #4] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:24,558 WARN [CommitterEvent Processor #4] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000000_1
2015-12-12 00:42:24,558 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_1 TaskAttempt Transitioned from FAIL_TASK_CLEANUP to FAILED
2015-12-12 00:42:24,564 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:24,566 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_2 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:42:24,627 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Added attempt_1449846414359_0019_m_000000_2 to list of failed maps
2015-12-12 00:42:24,634 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_1 TaskAttempt Transitioned from FAIL_CONTAINER_CLEANUP to FAIL_TASK_CLEANUP
2015-12-12 00:42:24,636 INFO [CommitterEvent Processor #0] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:24,693 WARN [CommitterEvent Processor #0] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000002_1
2015-12-12 00:42:24,694 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_1 TaskAttempt Transitioned from FAIL_TASK_CLEANUP to FAILED
2015-12-12 00:42:24,694 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:24,695 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_2 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:42:24,696 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Added attempt_1449846414359_0019_m_000002_2 to list of failed maps
2015-12-12 00:42:24,819 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Before Scheduling: PendingReds:1 ScheduledMaps:2 ScheduledReds:0 AssignedMaps:3 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:7 ContRel:1 HostLocal:3 RackLocal:0
2015-12-12 00:42:24,836 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=0 finishedContainers=0 resourcelimit=<memory:3072, vCores:-3> knownNMs=1
2015-12-12 00:42:24,836 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:3072, vCores:-3>
2015-12-12 00:42:24,836 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:24,974 INFO [IPC Server handler 26 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Progress of TaskAttempt attempt_1449846414359_0019_m_000001_1 is : 0.0
2015-12-12 00:42:24,994 FATAL [IPC Server handler 6 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Task: attempt_1449846414359_0019_m_000001_1 - exited : java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:24,995 INFO [IPC Server handler 6 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Diagnostics report from attempt_1449846414359_0019_m_000001_1: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:24,996 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000001_1: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:24,996 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_1 TaskAttempt Transitioned from RUNNING to FAIL_CONTAINER_CLEANUP
2015-12-12 00:42:24,999 INFO [ContainerLauncher #1] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000007 taskAttempt attempt_1449846414359_0019_m_000001_1
2015-12-12 00:42:25,000 INFO [ContainerLauncher #1] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000001_1
2015-12-12 00:42:25,000 INFO [ContainerLauncher #1] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:25,046 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_1 TaskAttempt Transitioned from FAIL_CONTAINER_CLEANUP to FAIL_TASK_CLEANUP
2015-12-12 00:42:25,049 INFO [CommitterEvent Processor #1] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:25,075 WARN [CommitterEvent Processor #1] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000001_1
2015-12-12 00:42:25,075 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_1 TaskAttempt Transitioned from FAIL_TASK_CLEANUP to FAILED
2015-12-12 00:42:25,076 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:25,077 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_2 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:42:25,140 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Added attempt_1449846414359_0019_m_000001_2 to list of failed maps
2015-12-12 00:42:25,836 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Before Scheduling: PendingReds:1 ScheduledMaps:3 ScheduledReds:0 AssignedMaps:3 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:7 ContRel:1 HostLocal:3 RackLocal:0
2015-12-12 00:42:25,859 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=1 finishedContainers=2 resourcelimit=<memory:4096, vCores:-2> knownNMs=1
2015-12-12 00:42:25,859 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000005
2015-12-12 00:42:25,860 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000006
2015-12-12 00:42:25,860 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000000_1: Container killed by the ApplicationMaster.

2015-12-12 00:42:25,860 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 1
2015-12-12 00:42:25,860 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000002_1: Container killed by the ApplicationMaster.
Container killed on request. Exit code is 143
Container exited with a non-zero exit code 143

2015-12-12 00:42:25,860 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigning container Container: [ContainerId: container_1449846414359_0019_01_000009, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] to fast fail map
2015-12-12 00:42:25,873 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned from earlierFailedMaps
2015-12-12 00:42:25,874 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000009 to attempt_1449846414359_0019_m_000000_2
2015-12-12 00:42:25,874 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:4096, vCores:-2>
2015-12-12 00:42:25,874 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:25,874 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:2 ScheduledReds:0 AssignedMaps:2 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:8 ContRel:1 HostLocal:3 RackLocal:0
2015-12-12 00:42:25,874 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:25,875 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_2 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:42:25,877 INFO [ContainerLauncher #0] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000009 taskAttempt attempt_1449846414359_0019_m_000000_2
2015-12-12 00:42:25,877 INFO [ContainerLauncher #0] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000000_2
2015-12-12 00:42:25,878 INFO [ContainerLauncher #0] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:25,901 INFO [ContainerLauncher #0] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000000_2 : 13562
2015-12-12 00:42:25,901 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000000_2] using containerId: [container_1449846414359_0019_01_000009 on NM: [ubuntu:49665]
2015-12-12 00:42:25,902 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_2 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:42:25,902 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000000
2015-12-12 00:42:26,963 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=1 finishedContainers=1 resourcelimit=<memory:4096, vCores:-2> knownNMs=1
2015-12-12 00:42:26,963 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000007
2015-12-12 00:42:26,963 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000001_1: Container killed by the ApplicationMaster.
Container killed on request. Exit code is 143
Container exited with a non-zero exit code 143

2015-12-12 00:42:26,963 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 1
2015-12-12 00:42:26,964 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigning container Container: [ContainerId: container_1449846414359_0019_01_000010, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] to fast fail map
2015-12-12 00:42:26,964 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned from earlierFailedMaps
2015-12-12 00:42:26,964 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:26,965 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_2 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:42:26,965 INFO [ContainerLauncher #3] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000010 taskAttempt attempt_1449846414359_0019_m_000002_2
2015-12-12 00:42:26,965 INFO [ContainerLauncher #3] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000002_2
2015-12-12 00:42:26,965 INFO [ContainerLauncher #3] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:26,982 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000010 to attempt_1449846414359_0019_m_000002_2
2015-12-12 00:42:26,983 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:4096, vCores:-2>
2015-12-12 00:42:26,983 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:26,983 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:1 ScheduledReds:0 AssignedMaps:2 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:9 ContRel:1 HostLocal:3 RackLocal:0
2015-12-12 00:42:27,030 INFO [ContainerLauncher #3] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000002_2 : 13562
2015-12-12 00:42:27,030 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000002_2] using containerId: [container_1449846414359_0019_01_000010 on NM: [ubuntu:49665]
2015-12-12 00:42:27,030 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_2 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:42:27,031 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000002
2015-12-12 00:42:28,024 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=1 finishedContainers=0 resourcelimit=<memory:3072, vCores:-3> knownNMs=1
2015-12-12 00:42:28,024 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 1
2015-12-12 00:42:28,024 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigning container Container: [ContainerId: container_1449846414359_0019_01_000011, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] to fast fail map
2015-12-12 00:42:28,024 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned from earlierFailedMaps
2015-12-12 00:42:28,025 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:28,025 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_2 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:42:28,025 INFO [ContainerLauncher #4] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000011 taskAttempt attempt_1449846414359_0019_m_000001_2
2015-12-12 00:42:28,025 INFO [ContainerLauncher #4] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000001_2
2015-12-12 00:42:28,025 INFO [ContainerLauncher #4] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:28,041 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000011 to attempt_1449846414359_0019_m_000001_2
2015-12-12 00:42:28,041 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:3072, vCores:-3>
2015-12-12 00:42:28,041 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:28,041 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:0 ScheduledReds:0 AssignedMaps:3 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:10 ContRel:1 HostLocal:3 RackLocal:0
2015-12-12 00:42:28,083 INFO [ContainerLauncher #4] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000001_2 : 13562
2015-12-12 00:42:28,084 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000001_2] using containerId: [container_1449846414359_0019_01_000011 on NM: [ubuntu:49665]
2015-12-12 00:42:28,084 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_2 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:42:28,084 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000001
2015-12-12 00:42:29,083 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=1 finishedContainers=0 resourcelimit=<memory:2048, vCores:-4> knownNMs=1
2015-12-12 00:42:29,083 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 1
2015-12-12 00:42:29,083 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Cannot assign container Container: [ContainerId: container_1449846414359_0019_01_000012, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] for a map as either  container memory less than required <memory:1024, vCores:1> or no pending map tasks - maps.isEmpty=true
2015-12-12 00:42:29,083 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:2048, vCores:-4>
2015-12-12 00:42:29,083 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:29,083 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:0 ScheduledReds:0 AssignedMaps:3 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:11 ContRel:2 HostLocal:3 RackLocal:0
2015-12-12 00:42:30,113 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=0 release= 1 newContainers=0 finishedContainers=1 resourcelimit=<memory:3072, vCores:-3> knownNMs=1
2015-12-12 00:42:30,114 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000012
2015-12-12 00:42:30,114 ERROR [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Container complete event for unknown container id container_1449846414359_0019_01_000012
2015-12-12 00:42:30,114 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:3072, vCores:-3>
2015-12-12 00:42:30,114 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:30,488 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:30,489 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:30,512 INFO [IPC Server handler 14 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000010 asked for a task
2015-12-12 00:42:30,512 INFO [IPC Server handler 14 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000010 given task: attempt_1449846414359_0019_m_000002_2
2015-12-12 00:42:30,542 INFO [IPC Server handler 12 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000009 asked for a task
2015-12-12 00:42:30,542 INFO [IPC Server handler 12 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000009 given task: attempt_1449846414359_0019_m_000000_2
2015-12-12 00:42:31,300 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:31,315 INFO [IPC Server handler 8 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000011 asked for a task
2015-12-12 00:42:31,316 INFO [IPC Server handler 8 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000011 given task: attempt_1449846414359_0019_m_000001_2
2015-12-12 00:42:34,839 INFO [IPC Server handler 26 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Progress of TaskAttempt attempt_1449846414359_0019_m_000002_2 is : 0.0
2015-12-12 00:42:34,839 INFO [IPC Server handler 6 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Progress of TaskAttempt attempt_1449846414359_0019_m_000000_2 is : 0.0
2015-12-12 00:42:34,862 FATAL [IPC Server handler 5 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Task: attempt_1449846414359_0019_m_000000_2 - exited : java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:34,864 INFO [IPC Server handler 5 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Diagnostics report from attempt_1449846414359_0019_m_000000_2: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:34,872 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000000_2: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:34,877 FATAL [IPC Server handler 8 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Task: attempt_1449846414359_0019_m_000002_2 - exited : java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:34,879 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_2 TaskAttempt Transitioned from RUNNING to FAIL_CONTAINER_CLEANUP
2015-12-12 00:42:34,888 INFO [IPC Server handler 8 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Diagnostics report from attempt_1449846414359_0019_m_000002_2: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:34,889 INFO [ContainerLauncher #5] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000009 taskAttempt attempt_1449846414359_0019_m_000000_2
2015-12-12 00:42:34,891 INFO [ContainerLauncher #5] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000000_2
2015-12-12 00:42:34,892 INFO [ContainerLauncher #5] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:34,894 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000002_2: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:34,896 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_2 TaskAttempt Transitioned from RUNNING to FAIL_CONTAINER_CLEANUP
2015-12-12 00:42:34,897 INFO [ContainerLauncher #6] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000010 taskAttempt attempt_1449846414359_0019_m_000002_2
2015-12-12 00:42:34,899 INFO [ContainerLauncher #6] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000002_2
2015-12-12 00:42:34,906 INFO [ContainerLauncher #6] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:34,988 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_2 TaskAttempt Transitioned from FAIL_CONTAINER_CLEANUP to FAIL_TASK_CLEANUP
2015-12-12 00:42:34,998 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_2 TaskAttempt Transitioned from FAIL_CONTAINER_CLEANUP to FAIL_TASK_CLEANUP
2015-12-12 00:42:34,999 INFO [CommitterEvent Processor #2] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:35,000 INFO [CommitterEvent Processor #3] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:35,010 WARN [CommitterEvent Processor #2] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000002_2
2015-12-12 00:42:35,020 WARN [CommitterEvent Processor #3] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000000_2
2015-12-12 00:42:35,021 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_2 TaskAttempt Transitioned from FAIL_TASK_CLEANUP to FAILED
2015-12-12 00:42:35,031 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:35,034 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_2 TaskAttempt Transitioned from FAIL_TASK_CLEANUP to FAILED
2015-12-12 00:42:35,042 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_3 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:42:35,044 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:35,051 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Added attempt_1449846414359_0019_m_000002_3 to list of failed maps
2015-12-12 00:42:35,052 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_3 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:42:35,052 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Added attempt_1449846414359_0019_m_000000_3 to list of failed maps
2015-12-12 00:42:35,132 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Before Scheduling: PendingReds:1 ScheduledMaps:2 ScheduledReds:0 AssignedMaps:3 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:11 ContRel:2 HostLocal:3 RackLocal:0
2015-12-12 00:42:35,144 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=0 finishedContainers=0 resourcelimit=<memory:3072, vCores:-3> knownNMs=1
2015-12-12 00:42:35,144 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:3072, vCores:-3>
2015-12-12 00:42:35,144 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:36,161 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000009
2015-12-12 00:42:36,161 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000010
2015-12-12 00:42:36,161 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000011
2015-12-12 00:42:36,161 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000000_2: Container killed by the ApplicationMaster.
Container killed on request. Exit code is 143
Container exited with a non-zero exit code 143

2015-12-12 00:42:36,161 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 1
2015-12-12 00:42:36,161 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000002_2: Container killed by the ApplicationMaster.
Container killed on request. Exit code is 143
Container exited with a non-zero exit code 143

2015-12-12 00:42:36,162 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigning container Container: [ContainerId: container_1449846414359_0019_01_000013, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] to fast fail map
2015-12-12 00:42:36,162 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned from earlierFailedMaps
2015-12-12 00:42:36,162 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_2 TaskAttempt Transitioned from RUNNING to FAIL_CONTAINER_CLEANUP
2015-12-12 00:42:36,162 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000001_2: Exception from container-launch.
Container id: container_1449846414359_0019_01_000011
Exit code: 1
Stack trace: ExitCodeException exitCode=1: 
	at org.apache.hadoop.util.Shell.runCommand(Shell.java:538)
	at org.apache.hadoop.util.Shell.run(Shell.java:455)
	at org.apache.hadoop.util.Shell$ShellCommandExecutor.execute(Shell.java:715)
	at org.apache.hadoop.yarn.server.nodemanager.DefaultContainerExecutor.launchContainer(DefaultContainerExecutor.java:211)
	at org.apache.hadoop.yarn.server.nodemanager.containermanager.launcher.ContainerLaunch.call(ContainerLaunch.java:302)
	at org.apache.hadoop.yarn.server.nodemanager.containermanager.launcher.ContainerLaunch.call(ContainerLaunch.java:82)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)


Container exited with a non-zero exit code 1

2015-12-12 00:42:36,162 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000013 to attempt_1449846414359_0019_m_000002_3
2015-12-12 00:42:36,163 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:5120, vCores:-1>
2015-12-12 00:42:36,163 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:36,163 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:1 ScheduledReds:0 AssignedMaps:1 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:12 ContRel:2 HostLocal:3 RackLocal:0
2015-12-12 00:42:36,164 INFO [ContainerLauncher #7] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000011 taskAttempt attempt_1449846414359_0019_m_000001_2
2015-12-12 00:42:36,174 INFO [ContainerLauncher #7] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000001_2
2015-12-12 00:42:36,174 INFO [ContainerLauncher #7] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:36,176 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:36,178 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_3 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:42:36,178 INFO [ContainerLauncher #8] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000013 taskAttempt attempt_1449846414359_0019_m_000002_3
2015-12-12 00:42:36,179 INFO [ContainerLauncher #8] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000002_3
2015-12-12 00:42:36,179 INFO [ContainerLauncher #8] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:36,199 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_2 TaskAttempt Transitioned from FAIL_CONTAINER_CLEANUP to FAIL_TASK_CLEANUP
2015-12-12 00:42:36,200 INFO [CommitterEvent Processor #4] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:36,205 WARN [CommitterEvent Processor #4] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000001_2
2015-12-12 00:42:36,213 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_2 TaskAttempt Transitioned from FAIL_TASK_CLEANUP to FAILED
2015-12-12 00:42:36,213 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:36,214 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_3 TaskAttempt Transitioned from NEW to UNASSIGNED
2015-12-12 00:42:36,215 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Added attempt_1449846414359_0019_m_000001_3 to list of failed maps
2015-12-12 00:42:36,241 INFO [ContainerLauncher #8] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000002_3 : 13562
2015-12-12 00:42:36,244 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000002_3] using containerId: [container_1449846414359_0019_01_000013 on NM: [ubuntu:49665]
2015-12-12 00:42:36,245 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_3 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:42:36,245 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000002
2015-12-12 00:42:37,163 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Before Scheduling: PendingReds:1 ScheduledMaps:2 ScheduledReds:0 AssignedMaps:1 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:12 ContRel:2 HostLocal:3 RackLocal:0
2015-12-12 00:42:37,216 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=1 finishedContainers=0 resourcelimit=<memory:4096, vCores:-2> knownNMs=1
2015-12-12 00:42:37,216 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 1
2015-12-12 00:42:37,219 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigning container Container: [ContainerId: container_1449846414359_0019_01_000014, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] to fast fail map
2015-12-12 00:42:37,219 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned from earlierFailedMaps
2015-12-12 00:42:37,220 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:37,222 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000014 to attempt_1449846414359_0019_m_000000_3
2015-12-12 00:42:37,223 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:4096, vCores:-2>
2015-12-12 00:42:37,223 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:37,225 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:1 ScheduledReds:0 AssignedMaps:2 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:13 ContRel:2 HostLocal:3 RackLocal:0
2015-12-12 00:42:37,226 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_3 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:42:37,228 INFO [ContainerLauncher #9] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000014 taskAttempt attempt_1449846414359_0019_m_000000_3
2015-12-12 00:42:37,228 INFO [ContainerLauncher #9] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000000_3
2015-12-12 00:42:37,229 INFO [ContainerLauncher #9] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:37,306 INFO [ContainerLauncher #9] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000000_3 : 13562
2015-12-12 00:42:37,307 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000000_3] using containerId: [container_1449846414359_0019_01_000014 on NM: [ubuntu:49665]
2015-12-12 00:42:37,307 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_3 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:42:37,307 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000000
2015-12-12 00:42:38,344 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 0 newContainers=2 finishedContainers=0 resourcelimit=<memory:2048, vCores:-4> knownNMs=1
2015-12-12 00:42:38,344 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 2
2015-12-12 00:42:38,344 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigning container Container: [ContainerId: container_1449846414359_0019_01_000015, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] to fast fail map
2015-12-12 00:42:38,344 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned from earlierFailedMaps
2015-12-12 00:42:38,345 INFO [AsyncDispatcher event handler] org.apache.hadoop.yarn.util.RackResolver: Resolved ubuntu to /default-rack
2015-12-12 00:42:38,345 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigned container container_1449846414359_0019_01_000015 to attempt_1449846414359_0019_m_000001_3
2015-12-12 00:42:38,346 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Assigning container Container: [ContainerId: container_1449846414359_0019_01_000016, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] to fast fail map
2015-12-12 00:42:38,346 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_3 TaskAttempt Transitioned from UNASSIGNED to ASSIGNED
2015-12-12 00:42:38,347 INFO [ContainerLauncher #2] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_LAUNCH for container container_1449846414359_0019_01_000015 taskAttempt attempt_1449846414359_0019_m_000001_3
2015-12-12 00:42:38,347 INFO [ContainerLauncher #2] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Launching attempt_1449846414359_0019_m_000001_3
2015-12-12 00:42:38,347 INFO [ContainerLauncher #2] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:38,409 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Releasing unassigned and invalid container Container: [ContainerId: container_1449846414359_0019_01_000016, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ]. RM may have assignment issues
2015-12-12 00:42:38,409 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:2048, vCores:-4>
2015-12-12 00:42:38,409 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:38,409 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:0 ScheduledReds:0 AssignedMaps:3 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:15 ContRel:3 HostLocal:3 RackLocal:0
2015-12-12 00:42:38,454 INFO [ContainerLauncher #2] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Shuffle port returned by ContainerManager for attempt_1449846414359_0019_m_000001_3 : 13562
2015-12-12 00:42:38,457 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: TaskAttempt: [attempt_1449846414359_0019_m_000001_3] using containerId: [container_1449846414359_0019_01_000015 on NM: [ubuntu:49665]
2015-12-12 00:42:38,458 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_3 TaskAttempt Transitioned from ASSIGNED to RUNNING
2015-12-12 00:42:38,458 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.speculate.DefaultSpeculator: ATTEMPT_START task_1449846414359_0019_m_000001
2015-12-12 00:42:39,523 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=1 release= 1 newContainers=1 finishedContainers=1 resourcelimit=<memory:2048, vCores:-4> knownNMs=1
2015-12-12 00:42:39,524 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000016
2015-12-12 00:42:39,524 ERROR [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Container complete event for unknown container id container_1449846414359_0019_01_000016
2015-12-12 00:42:39,525 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Got allocated containers 1
2015-12-12 00:42:39,525 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Cannot assign container Container: [ContainerId: container_1449846414359_0019_01_000017, NodeId: ubuntu:49665, NodeHttpAddress: ubuntu:8042, Resource: <memory:1024, vCores:1>, Priority: 5, Token: Token { kind: ContainerToken, service: 127.0.1.1:49665 }, ] for a map as either  container memory less than required <memory:1024, vCores:1> or no pending map tasks - maps.isEmpty=true
2015-12-12 00:42:39,526 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:2048, vCores:-4>
2015-12-12 00:42:39,526 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:39,526 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:1 ScheduledMaps:0 ScheduledReds:0 AssignedMaps:3 AssignedReds:0 CompletedMaps:0 CompletedReds:0 ContAlloc:16 ContRel:4 HostLocal:3 RackLocal:0
2015-12-12 00:42:39,729 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:39,751 INFO [IPC Server handler 26 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000013 asked for a task
2015-12-12 00:42:39,751 INFO [IPC Server handler 26 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000013 given task: attempt_1449846414359_0019_m_000002_3
2015-12-12 00:42:40,249 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:40,273 INFO [IPC Server handler 24 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000014 asked for a task
2015-12-12 00:42:40,274 INFO [IPC Server handler 24 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000014 given task: attempt_1449846414359_0019_m_000000_3
2015-12-12 00:42:40,564 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerRequestor: getResources() for application_1449846414359_0019: ask=0 release= 1 newContainers=0 finishedContainers=1 resourcelimit=<memory:3072, vCores:-3> knownNMs=1
2015-12-12 00:42:40,564 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Received completed container container_1449846414359_0019_01_000017
2015-12-12 00:42:40,564 ERROR [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Container complete event for unknown container id container_1449846414359_0019_01_000017
2015-12-12 00:42:40,564 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:3072, vCores:-3>
2015-12-12 00:42:40,564 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold not met. completedMapsForReduceSlowstart 1
2015-12-12 00:42:41,993 INFO [Socket Reader #1 for port 33520] SecurityLogger.org.apache.hadoop.ipc.Server: Auth successful for job_1449846414359_0019 (auth:SIMPLE)
2015-12-12 00:42:42,022 INFO [IPC Server handler 24 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID : jvm_1449846414359_0019_m_000015 asked for a task
2015-12-12 00:42:42,022 INFO [IPC Server handler 24 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: JVM with ID: jvm_1449846414359_0019_m_000015 given task: attempt_1449846414359_0019_m_000001_3
2015-12-12 00:42:43,952 INFO [IPC Server handler 24 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Progress of TaskAttempt attempt_1449846414359_0019_m_000000_3 is : 0.0
2015-12-12 00:42:43,977 FATAL [IPC Server handler 20 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Task: attempt_1449846414359_0019_m_000000_3 - exited : java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:43,978 INFO [IPC Server handler 20 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Diagnostics report from attempt_1449846414359_0019_m_000000_3: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:43,979 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000000_3: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:43,980 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_3 TaskAttempt Transitioned from RUNNING to FAIL_CONTAINER_CLEANUP
2015-12-12 00:42:43,981 INFO [ContainerLauncher #1] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000014 taskAttempt attempt_1449846414359_0019_m_000000_3
2015-12-12 00:42:43,982 INFO [ContainerLauncher #1] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000000_3
2015-12-12 00:42:43,982 INFO [ContainerLauncher #1] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:44,143 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_3 TaskAttempt Transitioned from FAIL_CONTAINER_CLEANUP to FAIL_TASK_CLEANUP
2015-12-12 00:42:44,161 INFO [CommitterEvent Processor #0] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:44,176 WARN [CommitterEvent Processor #0] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000000_3
2015-12-12 00:42:44,177 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000000_3 TaskAttempt Transitioned from FAIL_TASK_CLEANUP to FAILED
2015-12-12 00:42:44,508 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_m_000000 Task Transitioned from RUNNING to FAILED
2015-12-12 00:42:44,536 INFO [IPC Server handler 14 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Progress of TaskAttempt attempt_1449846414359_0019_m_000002_3 is : 0.0
2015-12-12 00:42:44,536 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: Num completed Tasks: 1
2015-12-12 00:42:44,543 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: Job failed as tasks failed. failedMaps:1 failedReduces:0
2015-12-12 00:42:44,555 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: job_1449846414359_0019Job Transitioned from RUNNING to FAIL_WAIT
2015-12-12 00:42:44,559 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_m_000001 Task Transitioned from RUNNING to KILL_WAIT
2015-12-12 00:42:44,559 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_m_000002 Task Transitioned from RUNNING to KILL_WAIT
2015-12-12 00:42:44,559 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_r_000000 Task Transitioned from SCHEDULED to KILL_WAIT
2015-12-12 00:42:44,560 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_3 TaskAttempt Transitioned from RUNNING to KILL_CONTAINER_CLEANUP
2015-12-12 00:42:44,560 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_3 TaskAttempt Transitioned from RUNNING to KILL_CONTAINER_CLEANUP
2015-12-12 00:42:44,562 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_r_000000_0 TaskAttempt Transitioned from UNASSIGNED to KILLED
2015-12-12 00:42:44,562 INFO [ContainerLauncher #0] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000015 taskAttempt attempt_1449846414359_0019_m_000001_3
2015-12-12 00:42:44,564 INFO [ContainerLauncher #0] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000001_3
2015-12-12 00:42:44,564 INFO [ContainerLauncher #0] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:44,566 INFO [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Processing the event EventType: CONTAINER_DEALLOCATE
2015-12-12 00:42:44,566 ERROR [Thread-50] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Could not deallocate container for task attemptId attempt_1449846414359_0019_r_000000_0
2015-12-12 00:42:44,571 INFO [ContainerLauncher #3] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: Processing the event EventType: CONTAINER_REMOTE_CLEANUP for container container_1449846414359_0019_01_000013 taskAttempt attempt_1449846414359_0019_m_000002_3
2015-12-12 00:42:44,576 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Before Scheduling: PendingReds:1 ScheduledMaps:0 ScheduledReds:0 AssignedMaps:3 AssignedReds:0 CompletedMaps:1 CompletedReds:0 ContAlloc:16 ContRel:4 HostLocal:3 RackLocal:0
2015-12-12 00:42:44,576 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_r_000000 Task Transitioned from KILL_WAIT to KILLED
2015-12-12 00:42:44,583 INFO [ContainerLauncher #3] org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl: KILLING attempt_1449846414359_0019_m_000002_3
2015-12-12 00:42:44,583 INFO [ContainerLauncher #3] org.apache.hadoop.yarn.client.api.impl.ContainerManagementProtocolProxy: Opening proxy : ubuntu:49665
2015-12-12 00:42:44,584 FATAL [IPC Server handler 26 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Task: attempt_1449846414359_0019_m_000002_3 - exited : java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:44,585 INFO [IPC Server handler 26 on 33520] org.apache.hadoop.mapred.TaskAttemptListenerImpl: Diagnostics report from attempt_1449846414359_0019_m_000002_3: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:44,591 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Recalculating schedule, headroom=<memory:3072, vCores:-3>
2015-12-12 00:42:44,595 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Reduce slow start threshold reached. Scheduling reduces.
2015-12-12 00:42:44,595 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: All maps assigned. Ramping up all remaining reduces:1
2015-12-12 00:42:44,595 INFO [RMCommunicator Allocator] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: After Scheduling: PendingReds:0 ScheduledMaps:0 ScheduledReds:1 AssignedMaps:3 AssignedReds:0 CompletedMaps:1 CompletedReds:0 ContAlloc:16 ContRel:4 HostLocal:3 RackLocal:0
2015-12-12 00:42:44,605 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: Diagnostics report from attempt_1449846414359_0019_m_000002_3: Error: java.io.IOException: Type mismatch in value from map: expected org.apache.hadoop.io.Text, received ChessDataA$IntArrayWritable
	at org.apache.hadoop.mapred.MapTask$MapOutputBuffer.collect(MapTask.java:1074)
	at org.apache.hadoop.mapred.MapTask$NewOutputCollector.write(MapTask.java:712)
	at org.apache.hadoop.mapreduce.task.TaskInputOutputContextImpl.write(TaskInputOutputContextImpl.java:89)
	at org.apache.hadoop.mapreduce.lib.map.WrappedMapper$Context.write(WrappedMapper.java:112)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:56)
	at ChessDataA$TokenizerMapper.map(ChessDataA.java:24)
	at org.apache.hadoop.mapreduce.Mapper.run(Mapper.java:145)
	at org.apache.hadoop.mapred.MapTask.runNewMapper(MapTask.java:784)
	at org.apache.hadoop.mapred.MapTask.run(MapTask.java:341)
	at org.apache.hadoop.mapred.YarnChild$2.run(YarnChild.java:163)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:415)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1628)
	at org.apache.hadoop.mapred.YarnChild.main(YarnChild.java:158)

2015-12-12 00:42:44,639 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_3 TaskAttempt Transitioned from KILL_CONTAINER_CLEANUP to KILL_TASK_CLEANUP
2015-12-12 00:42:44,640 INFO [CommitterEvent Processor #1] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:44,652 WARN [CommitterEvent Processor #1] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000001_3
2015-12-12 00:42:44,653 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000001_3 TaskAttempt Transitioned from KILL_TASK_CLEANUP to KILLED
2015-12-12 00:42:44,655 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_m_000001 Task Transitioned from KILL_WAIT to KILLED
2015-12-12 00:42:44,669 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_3 TaskAttempt Transitioned from KILL_CONTAINER_CLEANUP to KILL_TASK_CLEANUP
2015-12-12 00:42:44,671 INFO [CommitterEvent Processor #2] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: TASK_ABORT
2015-12-12 00:42:44,675 WARN [CommitterEvent Processor #2] org.apache.hadoop.mapreduce.lib.output.FileOutputCommitter: Could not delete hdfs://localhost:9000/user/parallels/chessdata/output_a/_temporary/1/_temporary/attempt_1449846414359_0019_m_000002_3
2015-12-12 00:42:44,676 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskAttemptImpl: attempt_1449846414359_0019_m_000002_3 TaskAttempt Transitioned from KILL_TASK_CLEANUP to KILLED
2015-12-12 00:42:44,679 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.TaskImpl: task_1449846414359_0019_m_000002 Task Transitioned from KILL_WAIT to KILLED
2015-12-12 00:42:44,686 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: job_1449846414359_0019Job Transitioned from FAIL_WAIT to FAIL_ABORT
2015-12-12 00:42:44,686 INFO [CommitterEvent Processor #3] org.apache.hadoop.mapreduce.v2.app.commit.CommitterEventHandler: Processing the event EventType: JOB_ABORT
2015-12-12 00:42:44,813 INFO [AsyncDispatcher event handler] org.apache.hadoop.mapreduce.v2.app.job.impl.JobImpl: job_1449846414359_0019Job Transitioned from FAIL_ABORT to FAILED
2015-12-12 00:42:44,816 INFO [Thread-86] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: We are finishing cleanly so this is the last retry
2015-12-12 00:42:44,816 INFO [Thread-86] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: Notify RMCommunicator isAMLastRetry: true
2015-12-12 00:42:44,816 INFO [Thread-86] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: RMCommunicator notified that shouldUnregistered is: true
2015-12-12 00:42:44,816 INFO [Thread-86] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: Notify JHEH isAMLastRetry: true
2015-12-12 00:42:44,816 INFO [Thread-86] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: JobHistoryEventHandler notified that forceJobCompletion is true
2015-12-12 00:42:44,816 INFO [Thread-86] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: Calling stop for all the services
2015-12-12 00:42:44,821 INFO [Thread-86] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: Stopping JobHistoryEventHandler. Size of the outstanding queue size is 0
2015-12-12 00:42:44,985 INFO [eventHandlingThread] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: Copying hdfs://localhost:9000/tmp/hadoop-yarn/staging/parallels/.staging/job_1449846414359_0019/job_1449846414359_0019_1.jhist to hdfs://localhost:9000/tmp/hadoop-yarn/staging/history/done_intermediate/parallels/job_1449846414359_0019-1449898908033-parallels-chess+taskA-1449898964543-0-0-FAILED-default-1449898913331.jhist_tmp
2015-12-12 00:42:45,105 INFO [eventHandlingThread] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: Copied to done location: hdfs://localhost:9000/tmp/hadoop-yarn/staging/history/done_intermediate/parallels/job_1449846414359_0019-1449898908033-parallels-chess+taskA-1449898964543-0-0-FAILED-default-1449898913331.jhist_tmp
2015-12-12 00:42:45,110 INFO [eventHandlingThread] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: Copying hdfs://localhost:9000/tmp/hadoop-yarn/staging/parallels/.staging/job_1449846414359_0019/job_1449846414359_0019_1_conf.xml to hdfs://localhost:9000/tmp/hadoop-yarn/staging/history/done_intermediate/parallels/job_1449846414359_0019_conf.xml_tmp
2015-12-12 00:42:45,139 INFO [eventHandlingThread] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: Copied to done location: hdfs://localhost:9000/tmp/hadoop-yarn/staging/history/done_intermediate/parallels/job_1449846414359_0019_conf.xml_tmp
2015-12-12 00:42:45,152 INFO [eventHandlingThread] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: Moved tmp to done: hdfs://localhost:9000/tmp/hadoop-yarn/staging/history/done_intermediate/parallels/job_1449846414359_0019.summary_tmp to hdfs://localhost:9000/tmp/hadoop-yarn/staging/history/done_intermediate/parallels/job_1449846414359_0019.summary
2015-12-12 00:42:45,155 INFO [eventHandlingThread] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: Moved tmp to done: hdfs://localhost:9000/tmp/hadoop-yarn/staging/history/done_intermediate/parallels/job_1449846414359_0019_conf.xml_tmp to hdfs://localhost:9000/tmp/hadoop-yarn/staging/history/done_intermediate/parallels/job_1449846414359_0019_conf.xml
2015-12-12 00:42:45,159 INFO [eventHandlingThread] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: Moved tmp to done: hdfs://localhost:9000/tmp/hadoop-yarn/staging/history/done_intermediate/parallels/job_1449846414359_0019-1449898908033-parallels-chess+taskA-1449898964543-0-0-FAILED-default-1449898913331.jhist_tmp to hdfs://localhost:9000/tmp/hadoop-yarn/staging/history/done_intermediate/parallels/job_1449846414359_0019-1449898908033-parallels-chess+taskA-1449898964543-0-0-FAILED-default-1449898913331.jhist
2015-12-12 00:42:45,162 INFO [Thread-86] org.apache.hadoop.mapreduce.jobhistory.JobHistoryEventHandler: Stopped JobHistoryEventHandler. super.stop()
2015-12-12 00:42:45,170 INFO [Thread-86] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Setting job diagnostics to Task failed task_1449846414359_0019_m_000000
Job failed as tasks failed. failedMaps:1 failedReduces:0

2015-12-12 00:42:45,175 INFO [Thread-86] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: History url is http://ubuntu:19888/jobhistory/job/job_1449846414359_0019
2015-12-12 00:42:45,190 INFO [Thread-86] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Waiting for application to be successfully unregistered.
2015-12-12 00:42:46,198 INFO [Thread-86] org.apache.hadoop.mapreduce.v2.app.rm.RMContainerAllocator: Final Stats: PendingReds:0 ScheduledMaps:0 ScheduledReds:1 AssignedMaps:3 AssignedReds:0 CompletedMaps:1 CompletedReds:0 ContAlloc:16 ContRel:4 HostLocal:3 RackLocal:0
2015-12-12 00:42:46,200 INFO [Thread-86] org.apache.hadoop.mapreduce.v2.app.MRAppMaster: Deleting staging directory hdfs://localhost:9000 /tmp/hadoop-yarn/staging/parallels/.staging/job_1449846414359_0019
2015-12-12 00:42:46,206 INFO [Thread-86] org.apache.hadoop.ipc.Server: Stopping server on 33520
2015-12-12 00:42:46,241 INFO [IPC Server listener on 33520] org.apache.hadoop.ipc.Server: Stopping IPC Server listener on 33520
2015-12-12 00:42:46,244 INFO [IPC Server Responder] org.apache.hadoop.ipc.Server: Stopping IPC Server Responder
2015-12-12 00:42:46,245 INFO [TaskHeartbeatHandler PingChecker] org.apache.hadoop.mapreduce.v2.app.TaskHeartbeatHandler: TaskHeartbeatHandler thread interrupted
